tests/unit/test_upgrade.py

```diff
@@ -115,7 +113,7 @@ def test_on_pgbouncer_pebble_ready(
 
         self.charm.upgrade._on_pgbouncer_pebble_ready(event)
 
-        _patch_port.assert_called_once_with(True)
+        _reconcile_k8s_service.assert_called_once()
```

Comment:
Better to `assert_called_once_with` with the expected args.

Reply:
addressed in 79f0de8362d1cb5f3d00125a778b4c39e1d5e91d